README.md

```diff
@@ -352,6 +352,7 @@
 - [SapperCommerce](https://github.com/itswadesh/sapper-ecommerce) - E-commerce storefront using Svelte & Tailwind CSS.
 - [Misiki Books](https://github.com/itswadesh/misiki-books) - Book shop using Vue + Moltin + Tailwind CSS.
 - [Ubuntu 20.04](https://github.com/vivek9patel/vivek9patel.github.io) - An Ubuntu desktop using React.js + Tailwind CSS.
+- [Angular Spotify](https://github.com/trungk18/angular-spotify) - Spotify client built with Angular, Nx Workspace and TailwindCSS.
```

Comment:
```suggestion
- [Angular Spotify](https://github.com/trungk18/angular-spotify) - Spotify client built with Angular, Nx Workspace and Tailwind CSS.
```

Reply:
@innocenzi thank you! good catch. I pushed the update.